src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -53,6 +58,9 @@ function MoneyRequestSelectorPage(props) {
         [CONST.IOU.MONEY_REQUEST_TYPE.SEND]: translate('iou.sendMoney'),
         [CONST.IOU.MONEY_REQUEST_TYPE.SPLIT]: translate('iou.splitBill'),
     };
+    const isFromGlobalCreate = !props.route.params.reportID;
```

Comment:
`reportID` is defined already
```suggestion
    const isFromGlobalCreate = !reportID;
```